rcp/org.testeditor.rcp4.views.teststepselection/src/org/testeditor/rcp4/views/teststepselector/TestStepSelector.xtend

```diff
@@ -79,7 +92,32 @@ class TestStepSelector {
 
 				})
 		]
+		val commandService = PlatformUI.workbench.getService(ICommandService)
+		commandService.addExecutionListener(new IExecutionListener() {
```

Comment:
I think we should extract this to a separate class. Also, the parameters of the methods should have proper names (see http://help.eclipse.org/luna/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fcore%2Fcommands%2FIExecutionListener.html)